src/extension/interop/ExtensionInterop.h

```diff
@@ -22,9 +22,12 @@ extern "C"
     /// \return The version number
     [[maybe_unused]] API_EXPORT int32_t ExtensionGetVersion(ThriveConfig* thriveConfig);
 
+    [[maybe_unused]] API_EXPORT bool ArrayMeshUnwrap(ArrayMeshVariant* mesh, float texelSize);
```

Comment:
As this method technically takes in just a godot variant, I think the opaqua handle should be called `GodotVariant` rather than `ArrayMeshVariant` as the object type is just `godot::Variant` and nothing specific to the mesh.